Code/Mantid/Framework/DataHandling/src/LoadIDFFromNexus.cpp

```diff
@@ -70,7 +81,15 @@ void LoadIDFFromNexus::exec() {
 
   // Look for parameter correction file
   std::string parameterCorrectionFile = getParameterCorrectionFile( localWorkspace->getInstrument()->getName() );
-  g_log.debug() << "Parameter correction file: " << parameterCorrectionFile;
+  g_log.notice() << "Parameter correction file: " << parameterCorrectionFile << "\n";
+
+  // Read parameter correction file, if found
+  std::string correctionParameterFile = "";
+  bool append = false;
+  if( parameterCorrectionFile != "") {
+    readParameterCorrectionFile( parameterCorrectionFile, "2015-07-23 12:00:00", correctionParameterFile, append );
```

Comment:
Above line 90 I recommend adding the comment: "If correction parameter file is found return it name as correctionParameterFile and whether this should be applied embedded parameters as an append action or replace action"